genetics.pde

```diff
@@ -108,7 +112,7 @@ void tickPopulation() {
       if (c.consumed > 230 && c.hydrated > 230) {
         while (critIt2.hasNext()) {
           Critter o = (Critter) critIt2.next();
-          if (o != c && o.consumed > 230 && o.hydrated > 230 && distance(o.x, o.y, c.x, c.y)< 5) {
+          if (o != c && o.consumed > 230 && o.hydrated > 230 && distance(o.x, o.y, c.x, c.y)< critterSize/2) { //chagned eat/drink distance to critterSize / 2
```

Comment:
Here we are detecting whether two critters are touching - if you're looking to accurately visually represent this, it would be `(o.size+c.size)/2`